src/software/jetson_nano/ansible/playbooks/deploy_nano.yml

```diff
@@ -5,20 +5,18 @@
   vars:
     all_services:
       - thunderloop
-      - wifi_announcement
     all_actions:
       - stop
       - sync
       - start
     binary_paths:
       thunderloop: ../../thunderloop_main
-      wifi_announcement: ../../broadcasts/robot_broadcast_sender_zip.zip
       display: ../../display/robot_ui_zip.zip
 
   tasks:
 
     - debug:
-        msg: "[Robot ID = {{ inventory_hostname }}]"
+          msg: "[Robot ID = {{ inventory_hostname }}]"
```

Comment:
curious if this was intended. I know yaml can be pretty sensitive with indentations, so wanted to double check.

Reply:
fixed